lib/topic.js

```diff
@@ -72,6 +74,44 @@ const removeAllTopicsRoles = (client, message) => {
   });
 }
 
+// We know that each inactive topic maps to a reaction according to the index
+// it is at in the topic array.
+const getReactionsOfInactives = (client, message) => {
+	let topics = client.topics;
+	let ind = 0;
+	// if the topic is active, we map it to undefined to filter it out later
+	// else, we map to its index
+	let indexOfInactives = topics.map(t => {
+		return t[1] ? null : ind++;
+	});
+	let filteredIndexOfInactives = indexOfInactives.filter(maybeNum => maybeNum != null);
+	// now we get the values of our reactions & make it an array to index easily
+	let reactions = message.reactions.cache.values();
+	let arrReactions = [...reactions];
+	let inactiveReactions = filteredIndexOfInactives.map(ind => arrReactions[ind]);
+	return inactiveReactions;
+}
+
+const removeReactionToInactiveTopic = (client, message) => {
```

Comment:
`Topics` instead of `Topic` since this function removes reactions for all inactive topics instead of just one.

Reply:
good catch